src/routes/trade/trade.jsx

```diff
@@ -222,12 +255,7 @@ const DisplayTick = (props) => {
       fallback={<Loader class={shared["loader-position"]} />}
     >
       <div class={styles["market-tick"]}>
-        <span
-          class={classNames(
-            dashboardStyles["badge"],
-            dashboardStyles[`badge--${difference().status}`]
-          )}
-        >
+        <span>
```

Comment:
Please check if the ticks are visible in dark theme as well.

Reply:
Yes I made changes now It is visible with dark theme too